packages/dds/map/src/test/mocha/directory.spec.ts

```diff
@@ -1108,7 +1184,7 @@ describe("Directory", () => {
 				);
 			});
 
-			it.skip("Directories should ensure eventual consistency using LWW approach 1", async () => {
+			it("Directories should ensure eventual consistency using LWW approach 1", async () => {
```

Comment:
Remove or update the comment in this test (and the subsequent test) now that you've fixed the buggy behaviour described in the comment.